www_src/fonts.less

```diff
@@ -0,0 +1,21 @@
+@fonts-path: '../fonts';
+
+@font-face {
+  font-family: 'Fira Sans';
+  src: local('Fira Sans Regular'),
+    url('@{fonts-path}/FiraSans-Regular.otf') format('otf'),
+    url('@{fonts-path}/FiraSans-Regular.woff') format('woff'),
```

Comment:
is this necessary for the platforms we're targeting? Every modern browser supports WOFF, there is no need anymore to fall back to "otf" and "ttf" versions "just in case" because that case won't arise. For android 4.2 and 4.3 WOFF is a problem, but then I would recommend falling back purely to the `otf` version. There is no need to also have `ttf` fallback.